test/addons-napi/test_constructor/test.js

```diff
@@ -13,7 +13,11 @@ assert.strictEqual(test_object.readwriteValue, 1);
 test_object.readwriteValue = 2;
 assert.strictEqual(test_object.readwriteValue, 2);
 
-assert.throws(() => { test_object.readonlyValue = 3; });
+const expected = new RegExp(
+  '^TypeError: Cannot assign to read only property \'readonlyValue\' of ' +
+  'object \'#<MyObject>\'$'
```

Comment:
Matching the complete error message seems fragile. What if V8 changes the wording of the error message?

Would matching the _type_ of error object be sufficient, using the overload of `assert.throws()` that takes a constructor?

```JavaScript
assert.throws(() => { /* some code that throws a TypeError */ }, TypeError);
```